packages/office-ui-fabric-react/src/components/Breadcrumb/Breadcrumb.base.tsx

```diff
@@ -99,20 +109,20 @@ export class Breadcrumb extends BaseComponent<IBreadcrumbProps, any> {
                     directionalHint: DirectionalHint.bottomLeftEdge
                   } }
                 />
-                { <Icon
+                <Divider
                   className={ css('ms-Breadcrumb-chevron', styles.chevron) }
                   iconName={ getRTL() ? 'ChevronLeft' : 'ChevronRight' }
-                /> }
+                />
               </li>
             ) }
             { renderedItems.map(
               (item, index) => (
                 <li className={ css('ms-Breadcrumb-listItem', styles.listItem) } key={ item.key || String(index) }>
                   { onRenderItem(item, this._onRenderItem) }
-                  <Icon
+                  { index !== lastItemIndex && <Divider
```

Comment:
you could just move this before the li, and say `index > 0`. Then you don't need a variable.